sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -595,6 +595,19 @@ class HiveDDLSuite
     }
   }
 
+  test("test show partitions") {
```

Comment:
can you call this something more specific, like `show partitions on non-existent table`